vscode/v2/coursera-submission/grading.py

```diff
@@ -0,0 +1,28 @@
+import json
+import re
+import requests
+
+COURSERA_SUBMISSION_URL = 'https://hub.coursera-apps.org/api/workspaceSubmissions.v1'
```

Comment:
Can you update this from `hub.coursera-apps.org` to `hub.labs.coursera.org` ? We have recently moved to a domain which is a subdomain of coursera.org. That avoids some security issues with iframing labs. The old domain will continue to function for a while but better to switch to a new domain

Reply:
Will do. Thanks!